packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -282,7 +282,7 @@ export class PivotBase extends BaseComponent<IPivotProps, IPivotState> {
       // React.Element<any> cannot directly convert to PivotItem.
       const item = React.Children.toArray(this.props.children)[index] as any;
 
-      if (typeof item === 'object' && item.type === PivotItem) {
+      if (typeof item === 'object' && item.type.name === PivotItem.name) {
```

Comment:
should be `item.type === PivotItemType`. The same fix was already applied in another part in this file. See #4867 and #4249

and changelog is missing (that's why the build fails -> `rush change`)